### PR TITLE
Update service hostnames

### PR DESCRIPTION
**Pull request by shepda**
Update service hostnames so specific services are called instead of being proxied

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,9 +129,11 @@ The following environment variables are required:
 2. AUTH_URL
 3. AUTH_REALM
 4. SESSION_SECRET
-5. WORKFLOW_URL
-6. PLATFORM_DATA_URL
-7. SESSION_NAME
+5. SESSION_NAME
+6. PROTOCOLÍ
```
Comment (pixie79) [inline README.md]: ??

File: docker-compose.yml [hunk 2]
```diff
@@ -6,9 +6,6 @@ services:
     restart: on-failure
     container_name: translation
     environment:
-      FORM_URL: "${SCHEME}://formio.${DOMAINNAME}"
```
Comment (pixie79) [inline docker-compose.yml]: Should these not be needed / updated ?

File: kube/deployment.yml [hunk 3]
```diff
@@ -125,12 +125,6 @@ spec:
           - name: enccerts
             mountPath: /enccerts
         env:
-        - name: FORM_URL
```
Comment (pixie79) [inline kube/deployment.yml]: I think these are still needed ?

File: test/controllers/formDataResolveControllerShift.test.js [hunk 4]
```diff
@@ -13,27 +13,27 @@ describe('Form Data Resolve Controller', () => {
...
-                .get('/api/platform-data/shift?email=eq.email')
+                .get('/shift?email=eq.email')
                 .reply(200, [{
                     locationid: 'currentlocationid',
                     teamid: 'teamid',
                     email: 'email'
                 }]);
             nock('http://localhost:9001')
-                .get('/api/platform-data/rf_location?locationid=eq.currentlocationid')
+                .get('/rf_location?locationid=eq.currentlocationid')
```
Comment (pixie79) [inline test/controllers/formDataResolveControllerShift.test.js]: no longer rf_

File: test/controllers/formDataResolveControllerShift.test.js [hunk 5]
```diff
@@ -13,27 +13,27 @@ describe('Form Data Resolve Controller', () => {
...
                 .reply(200, [{
                     locationname: 'Current',
                     locationid: 'currentlocationid',
                     bflocationtypeid: 'bflocationtypeid'
 
                 }]);
             nock('http://localhost:9001')
-                .get('/api/platform-data/rf_bflocationtype?bflocationtypeid=eq.bflocationtypeid')
+                .get('/rf_bflocationtype?bflocationtypeid=eq.bflocationtypeid')
```
Comment (pixie79) [inline test/controllers/formDataResolveControllerShift.test.js]: no longer rf_

File: test/forms.js [hunk 6]
```diff
@@ -285,7 +285,7 @@ const dataUrlForm = [
                         }
                     ],
                     "json": "",
-                    "url": "{$.environmentContext.platformDataUrl}/region",
+                    "url": "{$.environmentContext.operationalDataUrl}/region",
```
Comment (pixie79) [inline test/forms.js]: should be reference

File: README.md (hunk 1 above)
Comment (shepda) [reply README.md]: Fixed

File: docker-compose.yml (hunk 2 above)
Comment (shepda) [reply docker-compose.yml]: I updated and moved these urls to a config file so they're no longer needed here

File: test/controllers/formDataResolveControllerShift.test.js (hunk 4 above)
Comment (shepda) [reply test/controllers/formDataResolveControllerShift.test.js]: Fixed

File: test/controllers/formDataResolveControllerShift.test.js (hunk 5 above)
Comment (shepda) [reply test/controllers/formDataResolveControllerShift.test.js]: Fixed

File: test/forms.js (hunk 6 above)
Comment (shepda) [reply test/forms.js]: Fixed

File: docker-compose.yml (hunk 2 above)
Comment (pixie79) [reply docker-compose.yml]: This is whats sets the environment for when the container is running, without this where is your config file getting its environment file from?

Comment (pixie79) [reply docker-compose.yml]: This is whats sets the environment for when the container is running, without this where is your config file getting its environment file from?

**Pull request merged**